users/templates/users/profile.html

```diff
@@ -3,17 +3,21 @@
 {% block content %}
 
 <h2>{{ author.user.username }}</h2>
+<h2>{{ author.id }}</h2>
+<h2>{{ user.author.id }}</h2>
```

Comment:
Was this left in for debugging?

(also, according to the internet we should use `author.pk` rather than `author.id` and have been using it everywhere else)

Reply:
Oh yeah for debugging. Commited the wrong file here.
I'll make another pull request later tonight when I finish the profile page